js/main.js

```diff
@@ -1 +1,74 @@
+console.log("yo")
+
 /* Javascript goes here! */
+
+$(document).ready(function(){
+console.log('JQuery doing stuff');
+//eventually, this should be triggered by search button click/submit
+requestWeatherData();
+
+
+});
+
+function requestWeatherData(){
+  console.log('requesting data');
+  //this should request live weather data from the API
+  //eventually based on the user types
+  $("form").submit(function(event){
+  event.preventDefault();
+
+
+  $(".temperature").text('');
+  $(".humidity").text('');
+  $(".results-city").text('');
+  $(".results-flag").text('');
```

Comment:
Since you're doing the same thing here by clearing out the text, you can write this in one line with the jquery selectors comma separated.